### PR TITLE
Update the Pull_request_template.md file

### PR DESCRIPTION
**Pull request by Mariatta**
Changed the template to be inside an `<!--   -->` tag.
Let contributors know that the PR title should include bpo number.

### CODE REVIEW

**Comment by brettcannon** [comment]
The other option is to include a checklist that people can check off to make sure they have everything covered. Not sure if that's better than the comment since people will still need to change the body for the final commit, but it might help lead to people forgetting stuff less.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-!!! If this is a backport PR (PR made against branches other than `master`),
+<!--
+Thanks for your contribution!
+
+Please take a quick look at those things down there. They're quite important.
```
Comment (brettcannon) [inline .github/PULL_REQUEST_TEMPLATE.md]: "Please read this comment in its entirety. It's quite important."

**Comment by Mariatta** [comment]
Thanks for the quick review! 
Can I make the checklist inside the comment tag?
So we don't actually need to 'tick' each box.
Eventually the bots will tell them everything they need to know.

**Comment by brettcannon** [comment]
I don't think there's a point to making checklist if people can't physically check stuff off, so don't worry about it.

**Comment by Mariatta** [comment]
Ok. Let's try out this new PR template for now, we can think about the checklist later.

**Pull request merged**